pmoves/tools/secrets_local_hydrate.py

```
@@ -52,7 +53,17 @@ def _write_updates(env_path: Path, updates: Dict[str, str]) -> None:
         key = stripped.split("=", 1)[0].strip()
         index[key] = idx
 
+    validated_updates = {}
     for key, value in updates.items():
+        # Validate each value before writing
+        is_valid, error = validate_secret_value(key, value)
```

Ruff (0.15.10)

[warning] Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)